client/lib/feed-stream-store/test/index.js

```diff
@@ -1,3 +1,11 @@
+jest.mock( 'lib/analytics', () => ( {} ) );
+jest.mock( 'lib/data-poller', () => require( './lib/data-poller' ) );
+jest.mock( 'lib/post-normalizer', () => require( './lib/post-normalizer' ) );
+jest.mock( 'lib/wp', () => require( './lib/wp' ) );
+jest.mock( 'reader/stats', () => ( {
```

Comment:
@gwwar it's in progress https://github.com/Automattic/wp-calypso/pull/15577

for future ref, you can ping @Automattic/reader for reader stuff :)